scripts/check_frontend_coverage.py

```diff
@@ -597,6 +597,11 @@ def check_coverage_changes():
         total_lines = stanza.total_lines
         covered_lines = stanza.covered_lines
 
+        # Ignore files in core/tests/linter_tests/ directory, as these
+        # are test files used by linter.
+        if file_name.startswith(('invalid', 'valid')):
+            continue
```

Comment:
Can we avoid generating test for these files instead of ignoring the coverage report?

Reply:
I have no idea about this, maybe @marianazangrossi suggest something.

Reply:
@Hudda is this resolved?

Reply:
We need to use it, else frontend checks coverage is failing

Reply:
No, these files are generated by the third party import and if we want to avoid generate coverage then we have to add all these files in `NOT_FULLY_COVERED_FILENAMES` list which I don't think is a good idea.

Reply:
Hi all, I think a way to avoid it is to include these files in `combined-tests.spec.ts`. You can include it on the regex and then run frontend tests and check if it's being reported.